examples/common.h

```diff
@@ -31,6 +31,7 @@ struct gpt_params {
     int32_t main_gpu                        = 0;   // the GPU that is used for scratch and small tensors
     float   tensor_split[LLAMA_MAX_DEVICES] = {0}; // how split tensors should be distributed across GPUs
     bool    low_vram                        = 0;   // if true, reduce VRAM usage at the cost of performance
+    int32_t n_probs                         = 0;   // if greater than 1, output the probabilities of top n_probs tokens. Max 5
```

Comment:
Why 5?

Reply:
I was copying openai's api. I think it might be trying to prevent people from doing a `n_probs = 10000` and taking up ram on the server? 

But max 5 is not implemented in common.h but in server.cpp. I will correct the comments.